db/migrate/20210322132613_create_reminder_experiments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReminderExperiments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reminder_experiments do |t|
```

Comment:
Not using UUID for this model (and ReminderTemplate) because the id will need to be human usable through the command line and UUIDs are not great in that regard.

Reply:
We could use a different human-useable unique field if that's necessary. It might be valuable to keep UUIDs for:
- consistency (its the default for almost all our tables)
- globally uniqueness (hypothetically - comparing data across countries)

Reply:
You make a good point about global uniqueness. I hadn't considered it. I still haven't gotten used to the fact that we have multiple unrelated deployments. 

Maybe I'll add a unique friendly name or something. UUID is also manageable, but it could increase the likelihood of human error.

Reply:
> Maybe I'll add a unique friendly name or something

Yup, we have FriendlyId already in use in a bunch of other spots for `slug`, so that would work fine.